SwiftIsland/Map/MapPolygon.swift

```diff
@@ -28,27 +28,41 @@ final class MapPolygon: MKPolygonRenderer {
   override func draw(_ mapRect: MKMapRect, zoomScale: MKZoomScale, in context: CGContext) {
     super.draw(mapRect, zoomScale: zoomScale, in: context)
 
-    let paragraphStyle = NSMutableParagraphStyle()
-    paragraphStyle.alignment = .center
-    let text = "\(area.name)"
-    let font = UIFont.systemFont(ofSize: 50.0)
-    let attributes = [NSAttributedString.Key.paragraphStyle: paragraphStyle,
-                      NSAttributedString.Key.font: font,
-                      NSAttributedString.Key.foregroundColor: UIColor.themeColor(.redDark)]
-    let fontSize = text.size(withAttributes: attributes)
+    let attributes = textAttributes(with: UIFont.systemFont(ofSize: 50.0),
+                                    textAlignment: .center,
+                                    color: UIColor.themeColor(.redDark))
+    let text = NSString(string: "\(area.name)")
```

Comment:
We should not use `NSString`, we should use `String` instead. Since `String` is widely used in Swift and is a struct type, whereas `NSString` (although we can convert it seamlessly) comes from Objective-C and is a class, inheriting from `NSObject`. We should avoid using Objectiv-C types.
```suggestion
    let text = area.name
```

Reply:
Sure. This was an attempt at appeasing the type checker, since `size(withAttributes:)` is an `NSString` method and not a `String` method. Declaring it as the Objective-C version would smooth that out a bit.

Now that we're stretching the time limit anyway that isn't necessary anymore.